server/build/Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN apt-get update \
   poppler-utils \
   tidy \
   tzdata \
+  libheif \
```

Comment:
For someone not using Docker to run Mattermost code, like me, needs to manually install `libheif` on their system. I had to install via Homebrew to get it working. Would you be open to creating a docs PR as well to include this step in developer setup instructions?

Reply:
I wonder if there's a way to make that optional for development somehow. Most people aren't going to need to use or test this code while doing development, so it would be nice if we could make that step optional for development somehow